pom.xml

```diff
@@ -245,7 +245,11 @@ SOFTWARE.
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.1</version>
-          <configuration combine.self="override"/>
+          <configuration combine.self="override">
+            <compilerArgs>
+              <arg>-Xpkginfo:always</arg>
```

Comment:
@mximp I think this will break `jacoco` -- coverage information will stop collecting

Reply:
@yegor256 Why do you think so?

Reply:
@mximp my mistake, I mixed it up with this: https://github.com/objectionary/eo/blob/master/eo-runtime/pom.xml#L98 (surefire plugin, not compiler)